index.html

```diff
@@ -6,26 +6,45 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>My Portfolio</title>
     <link rel="stylesheet" href="CSS/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <script
       src="https://kit.fontawesome.com/1dea6ac3ad.js"
       crossorigin="anonymous"
     ></script>
   </head>
   <body>
     <header class="header">
-      <a href="#" id="logo">edith</a>
-      <nav>
+     <!-- <a href="#" id="logo">edith</a>-->
+      <nav class="navBar" id="navbar">
+        <a href="#header" class="navLogo">edith</a>
+       <ul class="navMenu leftArrow">
+          <li class="navItem">
+            <a href="#portfolio" class="navLink">Portfolio <span class="navSpan"></span></a>
+          </li>
+          <li class="navItem">
+            <a href="#about" class="navLink"
+              >About <span class="navAbout"></span></a
+            >
+          </li>
+          <li class="navItem">
+            <a href="#contact" class="navLink">Contact <span class="navSpan"></span></a>
+          </li>
+<li class="navItem">
+          </li>
+        </ul>
```

Comment:
- [ ] Kindly be noticed that you should follow the Figma design in the designing of mobile menu.
- [ ] Also notice that when the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Figma Design | Your Design
------------------ | ------------------
![Screenshot from 2022-06-21 16-09-48](https://user-images.githubusercontent.com/33752735/181123109-642c74cf-268f-4b37-a1d5-00e556e18eaf.png) | ![Screenshot from 2022-07-27 00-22-11](https://user-images.githubusercontent.com/33752735/181123137-2df37690-85dc-4dc3-ae0e-26e5eb8849f7.png)